package-lock.json

```diff
@@ -778,7 +778,7 @@
     "@ts-common/commonmark-to-markdown": {
       "version": "1.2.0",
       "resolved": "https://registry.npmjs.org/@ts-common/commonmark-to-markdown/-/commonmark-to-markdown-1.2.0.tgz",
-      "integrity": "sha1-Gy257ada2oNjbwcopc3sic0D08M=",
+      "integrity": "sha512-xFWpGZN1XzWN0egRs6gcT+tgYYw57us/Xr9euoaTsi9N+UR9ZxnG8Mrt4K/KlB54dNIED/LaGxReqRM5L7tKZA==",
```

Comment:
I'll leave to the swagger team to accept this, you might need to roll back this updated file - but nothing ARM related